src/api/v1beta1/properties.go

```diff
@@ -387,3 +399,10 @@ func splitArg(arg string) (key, value string) {
 	value = split[1]
 	return
 }
+
+func getImageTag(imageURI string) string {
+	if !strings.Contains(imageURI, ":") {
+		return ""
```

Comment:
If an image without a tag is supplied this would return nothing.
While technically correct, maybe we should instead go with the docker convention and assume that it's `latest`?